pubspec.yaml

```diff
@@ -0,0 +1,8 @@
+name: wCommon
+version: 0.1.1
+author: Rob Becker
+description: Dart bindings for wf-common
+dependencies:
+dev_dependencies:
+  unittest: <0.12.0
```

Comment:
Yeah, that's probably smart.  If someone runs pub downgrade right now, who knows what will happen.